### PR TITLE
[new release] omigrate (0.3.0)

### PR DESCRIPTION
**Pull request by tmattio**
Database migrations for Reason and OCaml

- Project page: <a href="https://github.com/tmattio/omigrate">https://github.com/tmattio/omigrate</a>
- Documentation: <a href="https://tmattio.github.io/omigrate/">https://tmattio.github.io/omigrate/</a>

##### CHANGES:

- Support partial migrations (tmattio/omigrate#4, @maiste)
- New Sqlite3 driver (tmattio/omigrate#3, @maiste)
- Support optional auth with psql driver (tmattio/omigrate#2, @maiste)

### CODE REVIEW

**Comment by dinosaure** [comment]
It seems that your package pick the `result` package instead of the `Stdlib.Result` module. You have two solution:
1) You can add the dependency `result {>= "1.5.0"}` into your OPAM file
2) or you can explicitely use `Stdlib.Result`.

**Comment by maiste** [comment]
I added the result upper bound as you suggested.

File: packages/omigrate/omigrate.0.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "uri"
+  "cmdliner" {>= "1.1.0"}
+  "logs"
+  "fmt"
+  "pgx"
+  "pgx_lwt_unix"
+  "sqlite3"
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (kit-ty-kate) [inline packages/omigrate/omigrate.0.3.0/opam:24]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/omigrate/omigrate.0.3.0/opam [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+maintainer: ["Thibaut Mattio"]
+authors: ["Thibaut Mattio"]
+license: "MIT"
+homepage: "https://github.com/tmattio/omigrate"
+doc: "https://tmattio.github.io/omigrate/"
+bug-reports: "https://github.com/tmattio/omigrate/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.08.0"}
+  "result" {>= "1.5.0"}
```
Comment (mseri) [inline packages/omigrate/omigrate.0.3.0/opam:13]:
```suggestion
  "result" {>= "1.5"}
```
With the .0 will not match any package

**Comment by mseri** [comment]
I've sent a PR for pgx: https://github.com/ocaml/opam-repository/pull/22254
I am just waiting to check if the incompatibility with ocaml 5.0 will only boil down to some dependencies or if we need to add an upper bound

**Comment by maiste** [comment]
Sure no problem! In the meantime, I opened an issue on `pgx` about it: https://github.com/arenadotio/pgx/issues/128

**Comment by mseri** [comment]
Thanks

**Pull request merged**